monai/networks/blocks/squeeze_and_excitation.py

```diff
@@ -74,18 +77,24 @@ def forward(self, x: torch.Tensor) -> torch.Tensor:
         """
         b, c = x.shape[:2]
         y: torch.Tensor = self.avg_pool(x).view(b, c)
-        y = self.fc(y).view([b, c] + [1] * (x.ndimension() - 2))
-        return x * y
+        y = self.fc(y).view([b, c] + [1] * (x.ndim - 2))
+        result = x * y
+
+        # Residual connection is moved here instead of providing an override of forward in ResidualSELayer since
+        # Torchscript has an issue with using super().
+        if self.add_residual:
+            result += x
+
+        return result
 
 
 class ResidualSELayer(ChannelSELayer):
     """
-    A "squeeze-and-excitation"-like layer with a residual connection::
+    A "squeeze-and-excitation"-like layer with a residual connection.
```

Comment:
this `::` is required for rendering the plot in fixed-width font

Reply:
Fixed, the warnings from sphinx were cryptic about the issue.